koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -405,10 +405,10 @@ def populate_distributed_cost_sql(self, start_date, end_date, provider_uuid, dis
         """
 
         key_to_file_mapping = {
-            metric_constants.PLATFORM_COST: "distribute_platform_cost.sql",
-            metric_constants.WORKER_UNALLOCATED: "distribute_worker_cost.sql",
-            metric_constants.STORAGE_UNATTRIBUTED: "distribute_unattributed_storage_cost.sql",
-            metric_constants.NETWORK_UNATTRIBUTED: "distribute_unattributed_network_cost.sql",
+            metric_constants.PLATFORM_COST: ("distribute_platform_cost.sql", False),
```

Comment:
is `populate_distributed_cost_sql` called without a cost model?

Reply:
Excellent point! I was testing it by calling update cost model costs which calls this function different than normal summary, back to the drawing board! Good catch @myersCody